core/server/themes/index.js

```diff
@@ -22,8 +23,17 @@ module.exports = {
         // Just read the active theme for now
         return themeLoader
             .loadOneTheme(activeThemeName)
-            .then(function activeThemeHasLoaded() {
-                debug('Activating theme (method A on boot)', activeThemeName);
+            .then(function activeThemeHasLoaded(theme) {
+                // Validate
+                return validate
```

Comment:
This always happens on bootstrap as far as i can see. The general use case is, that the active theme was already scanned. The edge case is, that somebody changes the db setting (`activeTheme`) manually.
Can't we not validate the active theme on boot strap?If no, doesn't this slow down the bootstrap again?
We have to load gscan and we do an active validation.